soc/xtensa/intel_adsp/common/CMakeLists.txt

```diff
@@ -99,3 +99,15 @@ add_custom_target(
       ${CMAKE_BINARY_DIR}/zephyr/main.mod
       ${CMAKE_BINARY_DIR}/zephyr/main.mod 2>${NULL_FILE}
 )
+
+if(CONFIG_BUILD_OUTPUT_STRIPPED)
+add_custom_command(
+  TARGET gen_modules POST_BUILD
+    COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+            $<TARGET_PROPERTY:bintools,strip_flag>
+            $<TARGET_PROPERTY:bintools,strip_flag_all>
+            $<TARGET_PROPERTY:bintools,strip_flag_infile>${CMAKE_BINARY_DIR}/zephyr/main.mod
+            $<TARGET_PROPERTY:bintools,strip_flag_outfile>${CMAKE_BINARY_DIR}/zephyr/stripped-main.mod # rename
```

Comment:
Maybe `main.mod.strip`? It might be easier to group files mentally in file listing.

Reply:
Thanks @dcpleung. I considered `main.mod.strip` for consistency but I think the `.strip` extension was a mistake. Stripping does not change the file type _at all_, this is very misleading.

I'm not suggesting to rename `zephyr.strip` to `stripped-zephyr.elf` (yet?) but I see a potential future where `zephyr.strip` is the outlier not renamed for backward compatibility and where all other stripped files are named (for instance) `stripped-boot.mod`.

Now still trying to address your concern: how about `main-stripped.mod`?

PS: I think `.mod` was a mistake too but changing that would be even more distruptive.

Reply:
`main.strip.mod`? `main-stripped.mod` is fine too.

Reply:
I went for the longer `main-stripped.mod` to reduce "collision risk" with other files.